src/public/interfaces.ts

```diff
@@ -129,6 +129,19 @@ export interface TeamInformation {
    */
   userTeamRole?: UserTeamRole;
 }
+
+/**
+ * Represents OS locale info used for formatting date and time data
+ */
+export interface LocaleInfo {
+  platform: string;
```

Comment:
I think we should support predefined enum list for platform

Reply:
changed the type to 'windows' | 'macos' to match globe library.